Resources/Locale/en-US/deltav/navmap-beacons/station-beacons.ftl

```diff
@@ -0,0 +1,28 @@
+station-beacon-epistemics = Epistemics
```

Comment:
Science beacon has been renamed, redundant

Reply:
This just moves it to our own DV file, so you can ignore the WD file

Reply:
Do you still want me to remove these changes in favor of the edits you made to the other file? I made them under the assumption this was the more favored method. But if you would rather not have them separated, it doesn't matter.

Reply:
If you're gonna move them to their own files, revert what I did in the base files and I'll accept it